code/controllers/subsystem/lighting.dm

```diff
@@ -89,6 +89,10 @@ var/datum/subsystem/lighting/SSlighting
 	else
 		changed_turfs.Cut()
 
+	if(config.starlight)
+		for(var/turf/space/S in world)
```

Comment:
Why not make this a bit faster by making new space tiles add themselves to a "space_turfs" list and have this loop that list